paddle/operators/bilinear_interp_op.cc

```diff
@@ -0,0 +1,81 @@
+/* Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserve.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License. */
+
+#include "paddle/operators/bilinear_interp_op.h"
```

Comment:
Why should we implement such an operator? 

If we implement a`bilinear_interp` operator, we also need to implement `最近邻插值`, `三次插值`, even those `gaussian blur` operators. So many operators will be an infinite workload to us.
In my mind, we only need to port the OpenCV base data structure represent `cv::Mat` to eigen matrix. And reuse OpenCV CPU/GPU function as kernel implement. 
And maybe put those operators into a `image` directory is better.

Reply:
`bilinear_interp` is an op what a user once suggested.  This op is not used to do data argumentation. It's used to do image segmentation. I don't think these operators `最近邻插值, 三次插值, even those gaussian blur` are needed now. And there is no need to port  the heavier OpenCV for this one operator now.  In addition, TensorFlow also implements this op: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/kernels/resize_bilinear_op.cc

Reply:
Okay, I see. 
I just notice that this operator implement encounter some problems. This operator is more likely for user specified not for the general purpose. 
I find that [Caffe2 Catalogue](https://caffe2.ai/docs/operators-catalogue.html) do not implement it at all. 
I just want to remind that we should implement most used operators since our deadline is looming.

Reply:
@dzhwinter Thank you for reminding!  Now some necessary operators could not be developed since the LoDTensor is not ready. We'll try our best to finish LoDTensor this week. And @luotao1 has several necessary sequence operators.